src/compiler/nodes.imba1

```diff
@@ -8830,9 +8830,10 @@ export class Tag < TagLike
 
 		let typ = isSelf ? "self" : (isFragment ? "'fragment'" : (type:isClass and type.isClass ? type.toTypeArgument : "'" + type.@value + "'"))
 		
-		if type.@value == 'global'
-			typ = "'i-global'"
-			STACK.use('dom_global_hook')
+		if type.@value == 'global' or type.@value == 'teleport'
+			console.warn "global will be deprecated in favor of teleport in a future version. Please use teleport instead" unless type.@value == 'teleport'
```

Comment:
I made teleport a generalized concept of the current global. So it makes sense to deprecate global at some point. Right?

We can even just remove it directly since we're in alpha.